x-pack/solutions/security/plugins/security_solution/server/endpoint/routes/resolver/entity/utils/security_modules.test.ts

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
Given that these are intentional change-detection tests, would it help to group them in a separate describe block like describe('data contract enforcement', ...) to make the intent clearer to future maintainers? Once we have this validated, maybe we could use imported values for checks instead of hardcoding them.